avocado/utils/asset.py

```diff
@@ -114,12 +114,13 @@ def _create_metadata_file(self, asset_file):
             with open(metadata_path, "w") as metadata_file:
                 json.dump(self.metadata, metadata_file)
 
-    def _download(self, url_obj, asset_path):
+    def _download(self, url_obj, asset_path, timeout=300):
```

Comment:
The timeout value should better be defined in this module, such as:

```python
#: The default timeout for the downloading of assets
DOWNLOAD_TIMEOUT=300
```

And then reused everywhere, including as the default value for the plugin.